VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/host-hearing-list/mocked-hearing-list.component.ts

```diff
@@ -0,0 +1,5 @@
+import { HostHearingListComponent } from './host-hearing-list.component-base';
+
+export class MockedHearingListComponent extends HostHearingListComponent {
+    retrieveHearingsForUser() {}
+}
```

Comment:
For something like this, where it's only used for testing the base component I think it's better to define this in the spec file.

Reply:
yea sure I can move it back, also lint appears to be complaing because I marked it as a directive an the name doesnt end with directive 😂

Reply:
I was wrong, it was more the base component not for this